src/pages/ElasticLegacy/PositionLegacy.tsx

```diff
@@ -0,0 +1,340 @@
+  border-bottom: 1px solid ${({ theme }) => theme.border};
+  font-size: 14px;
+`
+
+export default function PositionLegacy({ positions }: { positions: SubgraphPosition[] }) {
+  const { chainId, networkInfo, account } = useActiveWeb3React()
+  const { library } = useWeb3React()
+  const theme = useTheme()
+
+  const addresses = [...new Set(positions.map(item => [item.token0.id, item.token1.id]).flat())]
+
+  const tokenPrices = useTokenPrices(addresses)
+
+  const [feeRewards, setFeeRewards] = useState<{
+    [tokenId: string]: [string, string]
+  }>(() => positions.reduce((acc, item) => ({ ...acc, [item.id]: ['0', '0'] }), {}))
+
+  const multicallContract = useMulticallContract()
+
+  useEffect(() => {
+    const getData = async () => {
+      if (!multicallContract) return
+      const fragment = tickReaderInterface.getFunction('getTotalFeesOwedToPosition')
+      const callParams = positions.map(item => {
+        return {
+          target: (networkInfo as EVMNetworkInfo).elastic.tickReader,
+          callData: tickReaderInterface.encodeFunctionData(fragment, [
+            config[chainId].positionManagerContract,
+            item.pool.id,
+            item.id,
+          ]),
+        }
+      })
+
+      const { returnData } = await multicallContract?.callStatic.tryBlockAndAggregate(false, callParams)
+      setFeeRewards(
+        returnData.reduce(
+          (
+            acc: { [tokenId: string]: [string, string] },
+            item: { success: boolean; returnData: string },
+            index: number,
+          ) => {
+            if (item.success) {
+              const tmp = tickReaderInterface.decodeFunctionResult(fragment, item.returnData)
+              return {
+                ...acc,
+                [positions[index].id]: [tmp.token0Owed.toString(), tmp.token1Owed.toString()],
+              }
+            }
+            return { ...acc, [positions[index].id]: ['0', '0'] }
+          },
+          {} as { [tokenId: string]: [string, string] },
+        ),
+      )
+    }
+
+    getData()
+    // eslint-disable-next-line
+  }, [chainId, multicallContract, networkInfo, positions.length])
+
+  const [allowedSlippage] = useUserSlippageTolerance()
+  const deadline = useTransactionDeadline()
+
+  const [removeLiquidityError, setRemoveLiquidityError] = useState<string>('')
+  const [attemptingTxn, setAttemptingTxn] = useState(false)
+  const [txnHash, setTxnHash] = useState<string | undefined>()
+  const addTransactionWithType = useTransactionAdder()
+  const [showPendingModal, setShowPendingModal] = useState(false)
+
+  const handleDismiss = () => {
+    setShowPendingModal(false)
+    setTxnHash('')
+    setAttemptingTxn(false)
+    setRemoveLiquidityError('')
+  }
+
+  return (
+    <Wrapper>
+      <OverFlow>
+        <TableHeader>
+          <Text>NFT ID</Text>
+          <Text>POOLS</Text>
+          <Text>MY LIQUIDITY</Text>
+          <Text>MY FEES EARNED</Text>
+          <Text textAlign="right">ACTION</Text>
```

Comment:
trans

Reply:
legacy page does not need translation. will remove when user remove all liquidity